packages/config/src/projects/layer2s/orderly.ts

```diff
@@ -39,6 +41,63 @@ export const orderly: Layer2 = opStackL2({
     },
     activityDataSource: 'Blockchain RPC',
   },
+  nonTemplateEscrows: [
+    {
+      chain: 'arbitrum',
+      includeInTotal: false,
+      address: EthereumAddress('0x816f722424B49Cf1275cc86DA9840Fbd5a6167e9'),
+      sinceTimestamp: new UnixTime(1697682598),
+      tokens: ['USDC'],
+      source: 'external',
+      bridge: {
+        name: 'Arbitrum escrow -> LayerZero AMB',
+      },
+    },
+    {
+      chain: 'optimism',
+      includeInTotal: false,
+      address: EthereumAddress('0x816f722424B49Cf1275cc86DA9840Fbd5a6167e9'),
+      sinceTimestamp: new UnixTime(1701153879),
+      tokens: ['USDC'],
+      source: 'external',
+      bridge: {
+        name: 'Optimism escrow -> LayerZero AMB',
+      },
+    },
+    {
+      chain: 'base',
+      includeInTotal: false,
+      address: EthereumAddress('0x816f722424b49cf1275cc86da9840fbd5a6167e9'),
+      sinceTimestamp: new UnixTime(1712584295),
+      tokens: ['USDC'],
+      source: 'external',
+      bridge: {
+        name: 'Base escrow -> LayerZero AMB',
+      },
+    },
+    {
+      chain: 'mantle',
+      includeInTotal: false,
+      address: EthereumAddress('0x816f722424B49Cf1275cc86DA9840Fbd5a6167e9'),
+      sinceTimestamp: new UnixTime(1705831672),
+      tokens: ['USDC'],
+      source: 'external',
+      bridge: {
+        name: 'Mantle escrow -> LayerZero AMB',
+      },
+    },
+    {
+      chain: 'ethereum',
+      includeInTotal: false,
```

Comment:
why is this false here? is there any double counting with an ethereum escrow? or are there other reasons